README.md

```diff
@@ -1,155 +1,134 @@
 
-Collected installation recipes and notes may be found on [wiki pages](https://github.com/ChannelFinder/ChannelFinder-SpringBoot/wiki).
+### Manual Installation
 
 * Prerequisites
 
-  * JDK 17
-  * Elastic version 8.2.x
-  * <For authN/authZ using LDAP:> LDAP server, e.g. OpenLDAP
-
-* setup elastic search  
-  **Install**  
-  Download and install elasticsearch (verision 8.2.0) from [elastic.com](https://www.elastic.co/downloads/past-releases/elasticsearch-8-2-0)
-  following the instructions for your platform.\
-  <Alternatively:> Install the elastic server from your distribution using a package manager.  
-  
-* Build 
-```
-# Debian 10
-sudo apt-get install openjdk-17-jdk maven git curl wget
-wget https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-8.2.0-amd64.deb
-sudo dpkg -i elasticsearch-8.2.0-amd64.deb
-sudo systemctl start elasticsearch
-
-#### Checkout and build ChannelFinder service source
-git clone https://github.com/ChannelFinder/ChannelFinderService.git
-cd ChannelFinderService
-mvn clean install
+    * JDK 17
+    * Elastic version 8.11.x
+    * **For authN/authZ using LDAP:** LDAP server, e.g. OpenLDAP
 
-``` 
+#### Setup Elasticsearch
 
-#### Start the service  
+Options:
 
-* Using spring boot via Maven
+- Download and install elasticsearch (version 8.11.0)
+  from [elastic.com](https://www.elastic.co) following the instructions for
+  your platform.
+- Install the elastic server from your distribution using a package manager.
+- Run Elasticsearch in a [docker container](https://www.elastic.co/guide/en/elasticsearch/reference/current/docker.html)
 
-```
-mvn spring-boot:run
-```
+#### Running
 
-* or using the jar
+```bash
+sudo apt-get install openjdk-17-jre git curl wget
+sudo systemctl start elasticsearch # Or other command to run elastic search
 
-```
-java -jar target/ChannelFinder-4.7.0.jar
+# Replace verison with the release you want
+wget https://github.com/ChannelFinder/ChannelFinderService/releases/download/ChannelFinder-{version}/ChannelFinder-{version}.jar 
+java -jar target/ChannelFinder-*.jar
+``` 
+
+Other installation recipes can be found
+on [the wiki pages](https://github.com/ChannelFinder/ChannelFinder-SpringBoot/wiki).
+
+### Configuration
+
+By default, the channelfinder service will start on port 8080 with the default settings. To start with a
+different `application.properties` file:
+
+```bash
+java -Dspring.config.location=file:./application.properties -jar ChannelFinder-*.jar
 ```
 
-The above command will start the channelfinder service on port 8080 with the default settings,
-which use embedded ldap server with users and roles defined in the [`cf.ldif`](src/main/resources/cf.ldif) file.
+The default authentication includes an embedded ldap server with users and roles defined in
+the [`cf.ldif`](src/main/resources/cf.ldif) file.
 Note that `cf.ldif` contains **default credentials** and should only be used during testing and evaluation.
 
-#### Verification
+### Verification
```

Comment:
I had to add the extra curl verification steps due to requests from some EPICS developers...
Do you think there is a better way to incorporate them?

Reply:
I think we should consider the swagger documentation the way of looking for other queries to make. That way we aren't duplicating the api documentation in all of swagger, documentation and the readme.

Swagger in fact gives you the curl commands:
![image](https://github.com/user-attachments/assets/e95d613a-96cb-4004-9aac-95b95a98d049)

Reply:
Ok,
Can we add a note about the swagger, including its landing URL in the verification section.

Reply:
Done :)